pkg/kms/localkms/internal/util/localkms_aead.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package util
```

Comment:
should use a more descriptive package name.

Reply:
In Tink's Go integration package, AEAD is the AWS/GCP/Hashicorp KMS service. 

In aries-framework-go, this AEAD KMS service is used as a key wrapper utility to the overall framework KMS service which is itself manages the creation, rotation and fetching of keys.

`util` is used for a lack of a better name.. perhaps `keywrapper` would make more sense?

Reply:
I renamed it `tinkwrapper`